### DISCUSSION TITLE
MFA (2FA) for logins

### DISCUSSION THREAD

**Discussion by zhy-home-lab** [fuatakgun/eufy_security#722] [General]
Hi all,

I have attempted to start up the  Eufy Security Add-on but the logs indicated that it is pending 2FA. I did receive the code, but where do I input he 2FA code? The documentation is not as clear and I have searched everywhere to see if someone else had the same problem. 
shown below: 
> 2023-02-03 15:00:55.689  INFO  Eufy Security server listening on host 0.0.0.0, port 3000 
> 2023-02-03 15:00:56.161  INFO  Requested verification code for 2FA 

Update: after installing the integration thru HACs, I received the following: 
![image](https://user-images.githubusercontent.com/105765563/216640756-d250e213-d944-4fde-a974-1dfb62ab2b26.png)
![image](https://user-images.githubusercontent.com/105765563/216640855-689047a8-54cc-4906-a82b-ab99624ffae2.png)
I have tried to input the Captcha with both upper case and lower case (aVfH, avfh, AVFH), and they all report back as Unknown error occurred.

Any helps? Thanks!

**Comment by ebuzzz** [comment]
The authentication seems to be broken at the moment. A lot of people are having these issues.

**Comment by OnnoTimmermans** [comment]
I copied the 2Fa authentication code that I received per email into this field, after that my camera was added to HA.

**Comment by Silberfuxx** [comment]
I have the same Problem.
i also got the info that 2fa is needed in the Eufy Security Add-on and got an email with the 6 digit code, but i don't know where to put is. :-(
Also my Eufy Security Integrationsometimes it brings up the captcha, but after inserteing the letter an error occours.
Most of the time there is no captcha, but i think it's about too many requests. Then an error 500 Internal Server got itself in trouble is shown.
@Ohno-nl 
what field do you mean?
But i think you got past my problem with the Addon, because i dont see any cameras

**Comment by Silberfuxx** [comment]
![grafik](https://user-images.githubusercontent.com/122205149/223759656-212dee82-241c-452a-a72c-abeb513c1b03.png)

**Comment by Freezy80** [comment]
I suddenly have the same error. Requested message in the log.
And even an error with the captha that HA can't display the picture.

Any idea how to solve it? I even tried restoring a backup from two days ago but the error doesn't go away.
